coretex/networking/network_manager_base.py

```diff
@@ -246,6 +246,66 @@ def genericDownload(
 
         return response
 
+    def sampleDownload(
+        self,
+        endpoint: str,
+        destination: str,
```

Comment:
destination should be of type: `Union[Path, str]` and if `str` is passed convert it into a `Path` inside the function